File: responses/H3G23A-8H4/2_sum_of_certain_multiples.rb
# you can write to stdout for debugging purposes, e.g.
# puts "this is a debug message"

def solution(x)
  sum = 0
  (1...x).each do |element|
    next if element % 3 != 0 && element % 5 != 0
    next if element % 3 == 0 && element % 5 == 0
      
    sum += element
  end
  sum
end

File: responses/YTT5DJ-9VR/2_sum_of_certain_multiples.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(x):
    i = 0
    sum = 0
    while (i<x):
        sum+=6*(i+3) + 2*5 + 4 + 3*3 + 2 + 2
        i+=15
    
    i-=1
        
    #print(sum)
    
    while(i>=x):
        if i%5==0 or i%3==0:
            sum-=i
        i-=1
    
    return sum